apps/challenges/migrations/0019_added_leaderboard_data_table.py

```diff
@@ -10,7 +10,7 @@
 class Migration(migrations.Migration):
```

Comment:
@RishabhJain2018 : Can you give me some more context about this PR ? Why is the migration required and what is the current status ?

Reply:
While I was trying to give support for the challenges that use python3, I encountered the following error of migrations:
<img width="1440" alt="screen shot 2018-07-09 at 7 31 15 pm" src="https://user-images.githubusercontent.com/12206047/42455073-dcf6b172-83ae-11e8-8c5a-09947579cca7.png">
Then I researched on it and found that
- *Issue*: 
1. Python 3 detects the missing migration file in jobs file. (https://github.com/javrasya/django-river/issues/30) whereas Python2 doesn’t.
*Fixes Tried:*
1. I tried squashing all the migration files without manually changing the file(defected migration file) then it wasn’t getting squashed as it is throwing Node Not Found Error.
2. I tried squashing all the files by changing the migration file(defected migration file) manually but it created more dependency errors in the other apps.
3. I tried changing the defected migration file manually and it works like a charm. I know we should not change the migration file manually but in emergency we can do it (I think we can go with this ) (https://www.algotech.solutions/blog/python/django-migrations-and-how-to-manage-conflicts/, https://docs.djangoproject.com/en/1.10/howto/writing-migrations/).

@taranjeet Please suggest a more cleaner way to move ahead with this.